### PR TITLE
Edit note

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
-added edit feature

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
Looks Good

**Review by SK02K1 — APPROVED** [review]
The code is clean and self descriptive 
Greate work @mohammadkhizerkhan

File: src/components/note/Note.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    const { title, desc, id,isPinned } = note;
+    
+    return (
+      <>
+
+      <div class="note" key={id}>
+      <button class="btn primary-btn btn-icon pin-icon" onClick={()=>updatePin(note)}>
+            {isPinned ? <BsPinFill />:<BsPin/>}
+      </button>
+      <div class="title-div">
```
Comment (anuj-kumary) [inline src/components/note/Note.jsx]: change class to className

File: src/components/noteform/NoteForm.jsx [hunk 2]
```diff
@@ -1,14 +1,48 @@
...
 } from "react-icons/md";
-import {useNote} from "../../context/NoteContext"
 
-function NoteForm({closeForm}) {
-  const {noteInput,changeHandler,submitNote} = useNote();
-  
+import { addNote, updateNote } from "../../services/firebaseServices";
+
+function NoteForm({ closeForm, editNoteData, edit }) {
+  console.log(editNoteData);
```
Comment (anuj-kumary) [inline src/components/noteform/NoteForm.jsx]: Remove **console.log**

File: src/css/noteform.css [hunk 3]
```diff
@@ -6,7 +16,11 @@
     padding: 1.5rem;
     position: fixed;
     z-index: 4;
-    left: 22rem;
+    height: max-content;
+    /* top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0; */
```
Comment (anuj-kumary) [inline src/css/noteform.css]: Remove unwanted comment

**Review by anuj-kumary — APPROVED** [review]
- Code is clean.
- Good use of naming convention

Suggestion:-
- In some places, you write a class change to className
- before push code  remove **console.log**

**Comment by mohammadkhizerkhan** [comment]
> 

sure thanks @anuj-kumary

**Pull request merged**